test/e2e/cws-tests/tests/test_e2e_cspm_docker.py

```diff
@@ -56,27 +59,90 @@ def test_privileged_container(self):
 
             wait_agent_log(agent_name, self.docker_helper, CSPM_START_LOG)
 
-        with Step(msg="check agent event", emoji=":check_mark_button:"):
-            _, output = self.container.exec_run("security-agent compliance check --report")
-            findings = parse_output_and_extract_findings(output.decode(), [CSPM_RUNNING_DOCKER_CHECK_LOG])
-            self.finding = None
-            for f in findings:
-                if is_expected_docker_finding(f, self.container_id):
-                    self.finding = f
-            if self.finding is None:
-                raise LookupError(f"{agent_name} | {CSPM_RUNNING_DOCKER_CHECK_LOG}")
+        with Step(msg="check agent events", emoji=":check_mark_button:"):
+            self.container.exec_run("security-agent compliance check --dump-reports /tmp/reports.json --report")
+            _, output = self.container.exec_run("cat /tmp/reports.json")
+            findings = json.loads(output)
+
+            expected_findings = {
```

Comment:
I'm wondering if we shouldn't put this in a dedicated file called `cis_docker` that we import ? WDYT ?

Reply:
I thought about doing this, but we need the container_id for some rule so we cannot use a global constant